README.md

```diff
@@ -26,15 +26,15 @@ it into it's final memory location.
 
 This library allows you to do in-place initialization safely.
 
-### Nightly Needed for `alloc` and `std` features
+### Nightly Needed for `alloc` feature
 
-This library requires unstable features when the `alloc` or `std` features are enabled and thus
-can only be used with a nightly compiler. The internally used features are:
-- `allocator_api`
-- `get_mut_unchecked`
+This library requires the `allocator_api` unstable feature when the `alloc` feature
+is enabled and thus this feature can only be used with a nightly compiler.  The feature
```

Comment:
```suggestion
is enabled and thus this feature can only be used with a nightly compiler. The feature
```

Reply:
That was intentional but I guess it [shows my age](https://www.reddit.com/r/writing/comments/3xv8bt/when_and_why_did_we_suddenly_stop_putting_2/). :) Removed.